qiskit_ibm_runtime/api/rest/cloud_backend.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""IBM Cloud Backend REST adapter."""
+
+from typing import Dict, Any
+
+from .base import RestAdapterBase
+from ..session import RetrySession
+
+
+class CloudBackend(RestAdapterBase):
+    """Rest adapter for IBM Cloud backend related endpoints."""
+
+    URL_MAP = {
+        "configuration": "/configuration",
+        "properties": "/properties",
+        "pulse_defaults": "/defaults",
+        "status": "/status",
+    }
+
+    def __init__(
+        self, session: RetrySession, backend_name: str, url_prefix: str = ""
+    ) -> None:
+        """Backend constructor.
+
+        Args:
+            session: Session to be used in the adaptor.
+            backend_name: Name of the backend.
+            url_prefix: Base URL.
+        """
+        self.backend_name = backend_name
+        super().__init__(session, "{}/devices/{}".format(url_prefix, backend_name))
+
+    def configuration(self) -> Dict[str, Any]:
+        """Return backend configuration.
+
+        Returns:
+            JSON response of backend configuration.
+        """
+        url = self.get_url("configuration")
+        return self.session.get(url).json()
+
+    def properties(self) -> Dict[str, Any]:
+        """Return backend properties.
+
+        Returns:
+            JSON response of backend properties.
+        """
+        # pylint: disable=redefined-outer-name
```

Comment:
Are you sure this is needed? `make lint` does not fail for me without this.

Reply:
Good catch! Fixed here https://github.com/Qiskit/qiskit-ibm-runtime/pull/46/commits/f1b20231ec23eb22262b4b9d3c78ace4cd30ab82